src/serializers/styleSheetSerializer.js

```diff
@@ -38,8 +38,39 @@ const getMediaQueries = (ast, filter) => (
     }, [])
 )
 
-const getStyles = (classNames) => {
-  const styles = styleSheet.rules().map(rule => rule.cssText).join('\n')
+const resolveDeclarations = declarations => {
+  const declarationsMap = {}
+  return declarations
+    .reverse()
+    .filter(declaration => {
+      if (declarationsMap[declaration.property]) {
+        return false
+      }
+
+      declarationsMap[declaration.property] = true
+      return true
+    })
+    .reverse()
+}
+
+const getStyles = classNames => {
+  const styles = styleSheet
```

Comment:
This is not going to work with Styled Components v2, please check #14.
You should apply your changes to the AST.